docs/CreateAKSClusterForScalarDL.md

```diff
@@ -17,36 +17,50 @@ When deploying ScalarDL Ledger, you must:
 * Create the AKS cluster by using Kubernetes version 1.21 or higher.
 * Configure the AKS cluster based on the version of Kubernetes and your project's requirements.
 
-### Note
+{% capture notice--info %}
+**Note**
 
 For Byzantine fault detection in ScalarDL to work properly, do not deploy your application pods on the same AKS cluster as the ScalarDL Ledger deployment.
 
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
 ## Recommendations (optional)
 
 The following are some recommendations for deploying ScalarDL Ledger. These recommendations are not required, so you can choose whether or not to apply these recommendations based on your needs.
 
+### Create at least three worker nodes and three pods
+
+To ensure that the AKS cluster has high availability, you should use at least three worker nodes and deploy at least three pods spread across the worker nodes. You can see the [sample configurations](../conf/scalardl-custom-values.yaml) of `podAntiAffinity` for making three pods spread across the worker nodes.
+
+{% capture notice--info %}
+**Note**
+
+If you place the worker nodes in different [availability zones](https://learn.microsoft.com/en-us/azure/availability-zones/az-overview) (AZs), you can withstand an AZ failure.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
 ### Use 4vCPU / 8GB memory nodes for the worker node in the ScalarDL Ledger node pool
 
-From the perspective of commercial licenses, resources for one pod running ScalarDL Ledger are limited to 2vCPU / 4GB memory. In addition, we recommend deploying one ScalarDL Ledger pod and one Envoy pod on one worker node.
+From the perspective of commercial licenses, resources for one pod running ScalarDL Ledger are limited to 2vCPU / 4GB memory. In addition, some pods other than ScalarDL Ledger pods could exist on the worker nodes.
```

Comment:
```suggestion
From the perspective of commercial licenses, resources for one pod running ScalarDL Ledger are limited to 2vCPU / 4GB memory. In addition, some pods other than ScalarDL Ledger pods can exist on the worker nodes.
```

Are there any specific reasons to use the past tense? I feel 'can' is better.

Reply:
@jnmt  
In this sentence, what I want to tell users is `there is a possibility that some pods other than ScalarDL exist on the worker node`.

In other words, the main purpose of this sentence is NOT `we allow you to (you can) deploy some pods other than ScalarDL on the worker node`.

So, I use `could` as a `subjunctive mood` instead of `can`.

@josh-wong  
Sorry, I'm not confident about correct grammar...
What do you think?

Reply:
I understand what you mean here now. The following suggestion is similar to @jnmt's suggestion, with just some rewording for active voice and a couple more questions in parentheses:

```suggestion
From the perspective of commercial licenses, resources for one pod running ScalarDL Ledger are limited to 2vCPU / 4GB memory. In addition to the ScalarDL Ledger pod, Kubernetes (or the Kubernetes service?) could allocate (all or some of?) the following pods (or components?) to each worker node:
```

If we use a sentence structure similar to the above, I think we can remove the sentence (shown below) that follows this sentence.

```markdown
In other words, the following components could run on one worker node:
```

Reply:
@josh-wong  

Thank you for your new suggestion!

I updated the document as follows based on your suggestion in 986ceefd8e7647ad5407edfc04ee9a900acbd4e7.

* Ledger only

  ```console
  From the perspective of commercial licenses, resources for one pod running ScalarDL Ledger are limited to 2vCPU / 4GB memory. In addition to the ScalarDL Ledger pod, Kubernetes could deploy some of the following components to each worker node:
  ```

* Auditor mode

  ```console
  From the perspective of commercial licenses, resources for each pod running ScalarDL Ledger or ScalarDL Auditor are limited to 2vCPU / 4GB memory. In addition to the ScalarDL Ledger and ScalarDL Auditor pods, Kubernetes could deploy some of the following components to each worker node:
  ```

---

The following are the answer and reason of my updates.

> Kubernetes (or the Kubernetes service?)

Strictly, pod scheduling is done by the [`kube-scheduler`](https://kubernetes.io/docs/concepts/overview/components/#kube-scheduler). But, it's a bit internal thing. So, I think `Kubernetes` is simpler and better.

> allocate

I think `allocate` is not the wrong term. But, in the Kubernetes context, I think `deploy` is better.

> (all or some of?)

It's random things. It doesn't always deploy `all` components on one worker node. So, I think `some of` is better.

> following pods (or components?)

Strictly, there are some non-pod components such as [`kubelet`](https://kubernetes.io/docs/concepts/overview/components/#kubelet) that is one of the `Kubernetes components`. So, I think `components` is better.

---

Please take a look again when you have time!

Reply:
Sorry for the delay! LGTM😄